zebra/zebra_vxlan.c

```diff
@@ -2076,7 +2076,7 @@ static int zvni_macip_send_msg_to_client(vni_t vni, struct ethaddr *macaddr,
 
 	s = stream_new(ZEBRA_MAX_PACKET_SIZ);
 
-	zclient_create_header(s, cmd, VRF_DEFAULT);
+	zclient_create_header(s, cmd, zebra_vrf_get_evpn_id());
```

Comment:
In this file, Currently all L3 VNIs are handled only in default VRF. Add/Delete/Lookup is done only in default VRF. How are you planning to handle this ?

Reply:
Sorry, I think you already took care of this in other PR.

Reply:
(sorry for the resolve, finger slipped)

The other PR is unrelated to VRFs, but if I understand your comment correctly, the handling of L3VNIs is moved to the EVPN VRFs in commits 2118e6a9b4e875214fed275f628d4fa410d6f784 and cffe977c32cd9f73f00f43d9f0627e9c6813e259.

Reply:
I meant other commit that is part of the same PR.